### DISCUSSION TITLE
Please use the discussion forums

### DISCUSSION THREAD

**Discussion by pradeeban** [NISYSLAB/Emory-BMI-GSoC#13] [General]
Hi All,
Thanks a lot for your enthusiasm. This year, we have a large set of strong candidates. We thank you for all your contributions to Emory BMI projects such as CONTROL-CORE and Niffler.

One point I would like to note again is please make sure to use the relevant discussion forums as listed at https://github.com/NISYSLAB/Emory-BMI-GSoC/.

We keep receiving emails and even LinkedIn messages along the lines of "Can you please elaborate on what you want from this project? Any tips to become successful? How to get started?" We are ignoring those emails, and such contacts are not going to be a positive factor when we evaluate your project proposal (as they merely highlight the applicant did not follow through with the simple instructions in the README and is unlikely to be a great contributor).

If you already have a draft version of your proposal (as in a Google document) and like mentors' feedback, feel free to contact them via the email. But please note the mentors will comment only if they have a clarification to make or requests for changes/additions.

We will start reviewing the proposals officially on April 4th. As in the previous years, we are looking forward to yet another successful GSoC experience this year.

Thank you all again for your outstanding open source contributions. Please link to all the pull requests, discussions, and bug reports that you have created in your project proposal (even if they were closed as "Won't Fix" or "Invalid." We are evaluating the overall communication and effort. Not just the ones that are merged into the main (we cannot merge every fix without proper testing).

Happy coding!

**Comment by Yasien99** [comment]
Dear @pradeeban,

I hope this message finds you well. I wanted to reach out and apologize for any inconvenience caused by my previous emails regarding the Emory BMI GSoC projects. I understand that it is important to follow the instructions outlined in the README and to use the relevant discussion forums listed on GitHub.

Moving forward, I will ensure to communicate solely through GitHub discussions to avoid any miscommunication or confusion. I appreciate your patience and understanding.

I also wanted to inquire if my previous emails will have any negative impact on my project proposal evaluation. I have a draft version of my coding challenge part and I want to know if I can get any feedback to improve it, what is the right way to share it with my mentors?

I am interested in "Auto-detect coverage bounding boxes for brain MRI images" project.

Thank you for your time and guidance.

**Reply by pradeeban** [reply]
Hi and welcome, @Yasien99,
No, not at all. Your email won't leave a negative impact. We still receive several emails and messages (even through LinkedIn), so we don't remember those. Except when the email is of more private nature (for example, sharing your proposal draft as a Google doc link), we ignore the emails.

My point is more along the lines of using the time more effectively - both the applicants and mentors.

When you ask a question publicly, our replies are also public, and others can read and learn from them. Private emails are a waste of effort since we have to repeat the answers individually. We considered a mailing list. But realized GitHub discussions are a better approach to documenting the progress in a more open-source manner while minimizing email traffic.

Wishing you success in your GSoC applications. If you have specific questions, please feel free to ask any time in the relevant discussion channel. In your case, it is https://github.com/Emory-HITI/Niffler/discussions.
